app/src/components/ProgressBar.tsx

```diff
@@ -1,12 +1,25 @@
 import { FC } from 'react'
+import useOngoingTransferProgression from '@/hooks/useOngoingTransferProgression'
+import { StoredTransfer } from '@/models/transfer'
+import { SnowbridgeStatus } from '@/models/snowbridge'
+import { Direction } from '@/services/transfer'
 import { cn } from '@/utils/cn'
 
 interface ProgressBarProps {
-  progression: number
+  transfer: StoredTransfer
+  direction: Direction
   outlinedProgressBar: boolean
+  bridgeStatus?: SnowbridgeStatus
 }
 
-const ProgressBar: FC<ProgressBarProps> = ({ progression, outlinedProgressBar }) => {
+const ProgressBar: FC<ProgressBarProps> = ({
+  transfer,
+  direction,
+  bridgeStatus,
+  outlinedProgressBar,
+}) => {
+  const progression = useOngoingTransferProgression(transfer, direction, bridgeStatus)
```

Comment:
"progression" is not the right name here, we should rename this to "Progress" instead. Same goes to the hook naming


```suggestion
  const progress = useOngoingTransferProgression(transfer, direction, bridgeStatus)
```

Reply:
Sure, I will remove progression from everywhere and replace it by progress